service/src/main/java/org/mskcc/smile/service/impl/TempoServiceImpl.java

```diff
@@ -32,24 +38,22 @@ public class TempoServiceImpl implements TempoService {
     @Autowired
     private SmileRequestService requestService;
 
+    private static final Log LOG = LogFactory.getLog(TempoServiceImpl.class);
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ISO_LOCAL_DATE;
```

Comment:
This is the same time format we're using for the import date of sample and request metadata.